packages/cloudflare/src/index.ts

```diff
@@ -47,7 +47,7 @@ type Options = {
 	 * 'local': use a static req.cf object, and env vars defined in wrangler.toml & .dev.vars (astro dev is enough)
 	 * 'remote': use a dynamic real-live req.cf object, and env vars defined in wrangler.toml & .dev.vars (astro dev is enough)
 	 */
-	runtime?: 'off' | 'local' | 'remote';
+	runtime?: 'off' | 'local' | 'remote' | { mode: 'off' | 'local' | 'remote'; persistTo: string };
```

Comment:
Wondering how we could solve this. We want to use new API `{ mode: 'off' | 'local' | 'remote'; persistTo: string }` going forward, but still want to keep backwards compatibility, otherwise it might be a breaking change.